Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -273,29 +289,18 @@ - (void)showRewardedAdForParameters:(id<MAAdapterResponseParameters>)parameters
     NSString *placementIdentifier = parameters.thirdPartyAdPlacementIdentifier;
     [self log: @"Showing %@rewarded ad for placement: %@...", ( isBiddingAd ? @"bidding " : @"" ), placementIdentifier];
     
-    [self.router addShowingAdapter: self];
-    
-    NSError *error;
-    BOOL willShow = NO;
-    if ( isBiddingAd )
+    if ( self.vungleRewardedVideoAd && [self.vungleRewardedVideoAd canPlayAd] )
     {
-        if ( [[VungleSDK sharedSDK] isAdCachedForPlacementID: placementIdentifier adMarkup: bidResponse] )
+        UIViewController *presentingViewController;
+        if ( ALSdk.versionCode >= 11020199 )
         {
-            [self configureRewardForParameters: parameters];
-            willShow = [self showFullscreenAdForParameters: parameters error: &error];
+            presentingViewController = parameters.presentingViewController ?: [ALUtils topViewControllerFromKeyWindow];
         }
-    }
-    else if ( [[VungleSDK sharedSDK] isAdCachedForPlacementID: placementIdentifier] )
-    {
-        [self configureRewardForParameters: parameters];
-        willShow = [self showFullscreenAdForParameters: parameters error: &error];
-    }
-    
-    if ( !willShow || error )
-    {
-        MAAdapterError *adapterError = [MAAdapterError errorWithCode: -4205 errorString: @"Ad Display Failed" thirdPartySdkErrorCode: error.code thirdPartySdkErrorMessage: error.localizedDescription];
-        [self log: @"Rewarded ad failed to display with error: %@", adapterError];
-        [self.router didFailToDisplayAdForPlacementIdentifier: placementIdentifier error: adapterError];
+        else
+        {
+            presentingViewController = [ALUtils topViewControllerFromKeyWindow];
+        }
+        [self.vungleInterstitialAd presentWith: presentingViewController];
```

Comment:
Might it be `self.vungleInterstitialAd`-> `self.vungleRewardedVideoAd`? I saw this is for RewardedAd. Thanks a lot. :)